core/src/test/java/org/apache/accumulo/core/conf/PropertyTypeTest.java

```diff
@@ -18,38 +18,39 @@
  */
 package org.apache.accumulo.core.conf;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.lang.reflect.Method;
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TestName;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
 
 import com.google.common.base.Joiner;
 
+@DisplayNameGeneration(DisplayNameGenerator.Simple.class)
```

Comment:
Do we need to use this display name generator? What's wrong with the default?

Reply:
I figured it out. This was leftover from an earlier iteration of this where the display name was used instead of the test method name. I'll remove it. It's no longer needed, since we're not using the display name.